src/main/java/org/isf/accounting/rest/BillController.java

```diff
@@ -202,9 +202,10 @@ public ResponseEntity<FullBillDTO> updateBill(@PathVariable Integer id, @Request
 	 * @throws OHServiceException
 	 */
 	@GetMapping(value = "/bills", produces = MediaType.APPLICATION_JSON_VALUE)
-	public ResponseEntity<List<BillDTO>> searchBills(@RequestParam(value = "datefrom") @DateTimeFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'") @Schema(implementation = String.class) LocalDateTime dateFrom,
-			@RequestParam(value = "dateto") @DateTimeFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'") @Schema(implementation = String.class) LocalDateTime dateTo,
-			@RequestParam(value = "patient_code", required = false, defaultValue = "") Integer code) throws OHServiceException {
+	public ResponseEntity<List<BillDTO>> searchBills(
```

Comment:
also here we have a big space

Reply:
it's the formatter doing that (`...line_length" value="160"/>`)